LDAP-Auth/LDAPAuthenticationProviderPlugin.cs

```diff
@@ -113,14 +114,23 @@ public async Task<ProviderAuthenticationResult> Authenticate(string username, st
                         Array.Empty<string>(),
                         false);
 
-                    var foundUser = false;
-                    while (ldapUsers.HasMore() && !foundUser)
+                    if (EnableAdminFilterMemberUid)
+                    {
+                        ldapIsAdmin = ldapUsers.HasMore();
+                    }
+                    else
                     {
-                        var currentUser = ldapUsers.Next();
-                        if (string.Equals(ldapUser.Dn, currentUser.Dn, StringComparison.Ordinal))
+                        var foundUser = false;
+                        while (ldapUsers.HasMore() && !foundUser)
                         {
-                            ldapIsAdmin = true;
-                            foundUser = true;
+                            var currentUser = ldapUsers.Next();
+                            var dn = currentUser.Dn;
+                            _logger.LogWarning("Admin checking: {Dn}", dn);
+                            if (string.Equals(ldapUser.Dn, dn, StringComparison.Ordinal))
```

Comment:
```suggestion
                            if (string.Equals(ldapUser.Dn, currentUser.Dn, StringComparison.Ordinal))
```

I don't think the log line is needed

Reply:
My bad, fixed